open-api/rest-catalog-open-api.yaml

```diff
@@ -3324,6 +3348,211 @@ components:
           type: integer
           format: int64
 
+    BooleanTypeValue:
+      type: boolean
+      example: true
+
+    IntegerTypeValue:
+      type: integer
+      example: 42
+
+    LongTypeValue:
+      type: integer
+      format: int64
+      example: 9223372036854775807
+
+    FloatTypeValue:
+      type: number
+      format: float
+      example: 3.14
+
+    DoubleTypeValue:
+      type: number
+      format: double
+      example: 123.456
+
+    DecimalTypeValue:
+      type: string
+      description:
+        "Decimal types are serialized as strings using BigDecimal to maintain precision. For scales greater than 
+        or equal to zero, `toPlainString` ensures a non-exponential representation. Negative scales utilize the 
+       `toString`, resulting in exponential notation"
+      example:
+        positiveScale: "123.4500"
+        zeroScale: "2"
+        negativeScale: "2E+20"
+
+    StringTypeValue:
+      type: string
+      example: "hello"
+
+    UUIDTypeValue:
+      type: string
+      description:
+        "UUID types are serialized as a string"
+      example: "eb26bdb1-a1d8-4aa6-990e-da940875492c"
+
+    DateTypeValue:
+      type: string
+      format: date
+      description:
+        "Date types are serialized from an integer representing days since the Unix epoch to an ISO 8601 date string"
+      example: "2007-12-03"
+
+    TimeTypeValue:
+      type: string
+      description:
+        "Time types are serialized from a long representing microseconds since midnight to an ISO 8601 time string"
+      example: "10:15:30"
+
+    TimestampTypeValue:
+      type: string
+      description:
+        "Timestamp types are serialized from a long representing microseconds since the Unix epoch to an ISO 8601 
+        timestamp string. During serialization, the timestamp type is checked to see if it should be converted to UTC.
```

Comment:
"is checked to see if it should be converted to UTC" I guess you are talking about if it is timestamp with time zone vs timestamp without timezone?

Reply:
can we also add the format specification details? like YYYY-MM-DDTHH:MM:SS and YYYY-MM-DDTHH:MM:SS+00:00